lib/dialects/abstract/query-generator.js

```diff
@@ -1287,7 +1287,7 @@ class QueryGenerator {
             model
           },
           model
-        ).replace(/;$/, '') + ')';
+        ).replace(/;$/, '') + ') AS sub'; // Every derived table must have its own alias
```

Comment:
What if multiple includes with `separate:true` are used, wont having same `sub` alias throw an error? May be they are executed as separate statement so it will work.

Please extend your test case to demonstrate working with two includes with `separate: true` in same `findAll`